plugins/action/k8s_info.py

```diff
@@ -42,10 +42,15 @@ def run(self, tmp=None, task_vars=None):
         result = super(ActionModule, self).run(tmp, task_vars)
         del tmp  # tmp no longer has any effect
 
+        remote_transport = False
+        if self._connection.transport != 'local':
```

Comment:
Could you add some docs to the module describing exactly what the behavior users will see is? I don't know much about how Ansible handles all this stuff internally so I'm not really sure when it'll try to load it locally vs remotely

Reply:
Changing my review to approving because I might not get back to this in a reasonable amount of time, but still requesting some docs